app/src/test/java/org/simple/clinic/facility/change/confirm/ConfirmFacilityChangeUpdateTest.kt

```diff
@@ -10,13 +10,13 @@ import java.util.UUID
 
 class ConfirmFacilityChangeUpdateTest {
 
-  private val updateSpec = UpdateSpec<ConfirmFacilityChangeModel, ConfirmFacilityChangeEvent, ConfirmFacilityChangeEffect>(ConfirmFacilityChangeUpdate())
+  private val updateSpec = UpdateSpec(ConfirmFacilityChangeUpdate())
 
   @Test
   fun `when facility change is confirmed then user's facility should be changed`() {
     val selectedFacility = TestData.facility(UUID.fromString("ef47531f-9b8c-4045-8578-eda31f0952c4"))
     updateSpec
-        .given(ConfirmFacilityChangeModel())
+        .given(ConfirmFacilityChangeModel.create())
```

Comment:
~~This we are using the default model in a couple of tests, shall we extract these out into a global variable?~~ NVM this has been done in later commits.